bin/drush-install.sh

```diff
@@ -9,6 +9,7 @@
 ##################################
 ## List of extensions defining basic entity types
 ENTITY_EXTS=\
+uk.co.compucorp.civicrm.civihrcore,\
```

Comment:
This extension doesn't really define an entity, so it shouldn't be here (not that I think this differentiation is important, but still..)

You could add another group called `CORE_EXTS` which for now will include only this new extension, then you can do

```
drush "$@" cvapi extension.install keys=$CORE_EXTS,$ENTITY_EXTS,$APP_EXTS
```